test/fixtures/no-smart-oauth-uri-capability-statement.json

```diff
@@ -0,0 +1,228 @@
+{
+  "resourceType": "CapabilityStatement",
+  "status": "active",
+  "date": "2018-03-30T16:04:37+00:00",
+  "publisher": "Not provided",
+  "kind": "instance",
+  "software": {
+    "name": "HAPI FHIR Server",
+    "version": "2.4"
+  },
+  "implementation": {
+    "description": "HSPC Reference API Server"
+  },
+  "fhirVersion": "3.0.1",
+  "acceptUnknown": "extensions",
+  "format": [
+    "application/fhir+xml",
+    "application/fhir+json"
+  ],
+  "rest": [
+    {
+      "mode": "server",
+        "service": [
+          {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/restful-security-service",
+                "code": "SMART-on-FHIR",
+                "display": "SMART-on-FHIR"
+              }
+            ],
+            "text": "OAuth2 using SMART-on-FHIR profile (see http://docs.smarthealthit.org)"
+          }
+        ]
+      },
+      "resource": [
```

Comment:
This seems to have a syntax highlight problem (see line 226 of this file).

Is this a problem with github rendering or an actual problem?